tox.ini

```diff
@@ -18,6 +18,7 @@ deps =
     pytest
     pytest-httpbin
     PyYAML
+    multidict
```

Comment:
IMO this requirement is aiohttp only, so maybe it would better to tweak tox.ini by changing this line (as below) and moving it to the end of this section.

```ini
aiohttp: multidict
```